HCCGo/app/html/clusterLanding.html

```diff
@@ -54,9 +54,12 @@ <h4 class="usage text-center">/work Usage</h4>
 
     <div class="col-md-8 jobdisplay">
       <div class="card">
-        <input type="search" ng-model="q" placeholder="filter jobs..." />
+        <div class="form-control">
+          <input type="search" class="form-control" style="width:150px" placeholder="Filter Jobs..." />
```

Comment:
Sorry for creepin', but yeah inline styles hurt me, as do unnecessary static widths. Bootstrap grid classing would be good here, perhaps?